jade/page-contents/navbar_content.html

```diff
@@ -346,7 +346,9 @@ <h3 class="header">Navbar Dropdown Menu</h3>
         <pre><code class="language-javascript">
   document.addEventListener('DOMContentLoaded', function() {
     var elems = document.querySelectorAll('.dropdown-trigger');
-    var instances = M.Dropdown.init(elems, options);
+    var instances = M.Dropdown.init(elems, {
+      // specify options here
+    });
   });
 
   // Or with jQuery
```

Comment:
ehm you miss the jQuery one... also add the `$(document).ready` as well?

Reply:
nice catch! I've updated it :)